packages/core/review-workflows/admin/src/routes/settings/hooks/useReviewWorkflows.ts

```diff
@@ -31,7 +31,7 @@ const useReviewWorkflows = (params: UseReviewWorkflowsArgs = {}) => {
 
   const { data, isLoading, error } = useGetWorkflowsQuery(
     {
-      populate: 'stages',
+      populate: ['stages', 'stageRequiredForPublish'],
```

Comment:
Again just wording, but I think grammatically it makes more sense to have "stageRequiredToPublish"